flink-core/src/main/java/org/apache/flink/api/java/typeutils/runtime/TupleSerializer.java

```diff
@@ -59,7 +59,7 @@ public TupleSerializer(Class<T> tupleClass, TypeSerializer<?>[] fieldSerializers
 	@Override
 	public T createInstance() {
 		try {
-			T t = tupleClass.newInstance();
+			T t = tupleClass.getDeclaredConstructor().newInstance();
```

Comment:
this change may have performance implications as noted in the mail thread linked in the jira.